test/support/helpers/file_tags_helper.rb

```diff
@@ -0,0 +1,9 @@
+module FileTagsHelper
+  extend ActiveSupport::Concern
+
+  included do
+    def strip_file_type(filename:)
```

Comment:
I don't love the arbitrary splitting (and then only matching on the non-extension component).

Instead, I'd love to see a first-class matcher for images that works with Active Storage. Something like this could work.

```ruby
def assert_tag_source(tag: "img", filename:)
  components = filename.split(".")
  page.find(tag)["src"].match? /#{components.first}.*\.#{components.last}$/
end
```

Reply:
We needed an assertion in the method in order to get it to properly fail when giving in a bad input. I tested it out with something I knew would fail, and what I pushed to my branch works appropriately.

I appreciate your point about writing a first class assertion method rather than a bandage method that's super specific to my use case. It's a great feeling knowing you pay attention to things like that when reviewing these PRs.